internal/api/websocket.go

```diff
@@ -0,0 +1,73 @@
+package api
+
+import (
+	"context"
+	"github.com/google/uuid"
+	"github.com/gorilla/websocket"
+	core "github.com/iden3/go-iden3-core"
+	"github.com/polygonid/sh-id-platform/internal/core/domain"
+	"github.com/polygonid/sh-id-platform/internal/core/ports"
+	"log"
+	"net/http"
+	"time"
+)
+
+type WebsocketResponse struct {
+	ctx             context.Context
+	request         SubscribeToClaimWebsocketRequestObject
+	claimService    ports.ClaimsService
+	identityService ports.IdentityService
+}
+
+func (wr WebsocketResponse) VisitSubscribeToClaimWebsocketResponse(w http.ResponseWriter) error {
+	var upgrader = websocket.Upgrader{}
+
+	value := wr.ctx.Value("req-req")
```

Comment:
It makes sense to move `"req-req"` to constant with a description of why we need it